lib/middleware/response-entitlements.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+function yes() {
+	return true;
+}
+
+module.exports = function () {
+	return function responseEntitlements(req, res, next) {
+		const channel = req.identity.channel;
+		const viewer = req.identity.viewer;
+
+		// Default the evaluator to return true
+		let entitledEvaluator = yes;
+
+		// If auth is enabled then reset the evaluator to the one set on the channel
+		if (channel.features.authentication.enabled) {
+			entitledEvaluator = eval(`(${channel.features.authentication.evaluator})`); // eslint-disable-line no-eval
```

Comment:
Yeah. I was thinking of a nice way to handle that. We will be the ones writing these for now, but that would be a good safety net. If it fails we should probably default `entitlements = false` then.